content/docs/ops/maintenance-list.md

```diff
@@ -29,7 +29,7 @@ The tasks on this checklist should be performed each day.
 
 ## Review and respond to open alerts
 
-Review all recent alerts and notifications delivered to `cg-notifications` and `#cloud-gov-atlas-news`.  
+Review all recent alerts and notifications delivered to `cg-notifications` and [`#cg-platform-news`](https://gsa-tts.slack.com/messages/cg-platform-news/).
```

Comment:
whoops. sorry this was super unclear to me at first too. `cg-notifications` is the Google Group we use for emailed notifications in place of PagerDuty. @brittag clearly this needs to be a little more clear.